backend-schedule/src/main/java/eu/socialedge/hermes/backend/schedule/domain/ScheduleSpecification.java

```diff
@@ -27,11 +27,8 @@
 @Getter @Accessors(fluent = true)
 public class ScheduleSpecification {
 
-    @NotNull(message = "Inbound route id must not be null")
-    private Long inboundRouteId;
-
-    @NotNull(message = "Outbound route id must not be null")
-    private Long outboundRouteId;
+    @NotNull(message = "Line id must not be null")
+    private Long lineId;
```

Comment:
You should not use id to resource here since there is no way to retrieve it. /lines/1 will return no id = 1, only link to resource:
```
"line": {
   "href": "http://localhost:8080/lines/1"
}
```

Consider passing link here.